### PR TITLE
build: install liburing explicitly on ArchLinux.

### PR DESCRIPTION
**Pull request by balusch**
Signed-off-by: Jianyong Chen <baluschch@gmail.com>

### CODE REVIEW

File: src/core/reactor.cc [hunk 1]
```diff
@@ -1922,7 +1922,7 @@ reactor::spawn(std::string_view pathname,
                                                       argv = std::move(argv),
                                                       env = std::move(env), this] (std::tuple<file_desc, file_desc> cin_pipe,
                                                                                    std::tuple<file_desc, file_desc> cout_pipe,
-                                                                                   std::tuple<file_desc, file_desc> cerr_pipe) {
+                                                                                   std::tuple<file_desc, file_desc> cerr_pipe) mutable {
```
Comment (tchaikov) [inline src/core/reactor.cc:1925]: thanks! this might address #1320.

File: src/core/reactor.cc [hunk 2]
```diff
@@ -481,7 +481,7 @@ future<size_t> pollable_fd_state::recvmsg(struct msghdr *msg) {
         speculate_epoll(EPOLLIN);
         return make_ready_future<size_t>(*r);
     });
-};
+}
```
Comment (tchaikov) [inline src/core/reactor.cc:484]: i'd suggest s/style/reactor/ in the title of the commit message.

Comment (balusch) [reply src/core/reactor.cc:484]: good idea, the word `style` seems too general.

**Comment by avikivity** [comment]
In the future, please don't mix unrelated changes in the same series.

**Comment by balusch** [comment]
@avikivity  Thanks, avi.

Yes, its not a good habit(I just cannot help making more changes while seeing other issues. 😄), and I'll pay attention to that.

**Pull request merged**